src/handler.ts

```diff
@@ -264,8 +264,6 @@ export interface HandlerOptions<
  *     console.error(err);
  *     // or
  *     Sentry.captureException(err);
- *
- *     res.writeHead(500, 'Internal Server Error').end();
```

Comment:
If the error is thrown _before_ an SSE connection is established, the header is then still writable. Only during the execution phase inside an active SSE connection is the header already flushed.

```suggestion
 *     if (!res.headersSent) {
 *       res.writeHead(500, 'Internal Server Error').end();
 *     } 
```